### PR TITLE
Add Sentry integration, version 2

### PR DESCRIPTION
**Pull request by spladug**
The previous attempt used the raven-provided middleware. Unfortunately,
we have to run code in the wsgi handler context as well and it gets
application state from the application so things got rather complex when
the application was wrapped in a middleware. This is a less invasive but
more manual way of adding raven into the mix.

### CODE REVIEW

File: reddit_service_websockets/socketserver.py [hunk 1]
```diff
@@ -131,6 +134,22 @@ def __init__(self, metrics,
         self.connections = set()
 
     def __call__(self, environ, start_response):
+        try:
+            return self._handle_request(environ, start_response)
+        except (KeyboardInterrupt, SystemExit):
+            raise
+        except:
+            self.error_reporter.http_context({
```
Comment (ckwang8128) [inline reddit_service_websockets/socketserver.py]: do we care that self.error_reporter could technically be `None` here (e.g. in tests below)?

**Comment by ckwang8128** [comment]
Agreed that while this is a bit more manual it definitely seems cleaner. :nail_care: for small question about `None`-ness of `error_reporter`

**Comment by spladug** [comment]
:haircut_man:

**Comment by ckwang8128** [comment]
:fish:

**Comment by spladug** [comment]
8acd0f483b8c61be7b31c37c626d6b558fd6d8e0

**Pull request closed without merging**